composer.json

```diff
@@ -12,8 +12,8 @@
     ],
     "require": {
         "php": "^7.0.8",
-        "commerceguys/addressing": "^0.8",
-        "commerceguys/intl": "^0.7",
+        "commerceguys/addressing": "^1",
+        "commerceguys/intl": "^1",
```

Comment:
As far as I can see, after those changes, addressing-bundle doesn't depend directly on `commerceguys/intl` anymore, so that dependency can be dropped ?

Reply:
Agreed